packages/salesforcedx-vscode-soql/src/editor/htmlUtils.ts

```diff
@@ -84,7 +84,7 @@ export class HtmlUtils {
   ): string {
     const cspMetaTag = `<meta
       http-equiv="Content-Security-Policy"
-      content="default-src 'none';
+      content="default-src 'self';
```

Comment:
I thought the CSP was preventing scripts for the table view form being loaded during testing, it may have just been a race condition but I think adding `self` is more secure anyway. I'm not a CSP expert though.